Description: Accepts latest oslo.messaging
 Upstream is too restrictive with the accepted version of oslo.messaging.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-04-13

--- glance-2015.1~rc1.orig/requirements.txt
+++ glance-2015.1~rc1/requirements.txt
@@ -50,7 +50,7 @@ six>=1.9.0
 oslo.db>=1.7.0,<1.8.0  # Apache-2.0
 oslo.i18n>=1.5.0,<1.6.0  # Apache-2.0
 oslo.log>=1.0.0,<1.1.0  # Apache-2.0
-oslo.messaging>=1.8.0,<1.9.0  # Apache-2.0
+oslo.messaging>=1.8.0
 oslo.policy>=0.3.1,<0.4.0  # Apache-2.0
 oslo.serialization>=1.4.0,<1.5.0               # Apache-2.0
 
